gradle/lib/dependabot/gradle/update_checker/version_finder.rb

```diff
@@ -97,7 +97,7 @@ def filter_ignored_versions(possible_versions)
           filtered = possible_versions
 
           ignored_versions.each do |req|
-            ignore_req = Gradle::Requirement.new(req.split(","))
+            ignore_req = Gradle::Requirement.new(req)
```

Comment:
@eager thanks for this! We'll still need to support the old-style ruby syntax which exists in a bunch of config files but we should be able to support both.

@jurre looks like we might need to include your fix for gradle/maven/nuget requirements in core too.

Reply:
Ah nice one 🎉 I'll try to port it over, @eager if you're up for it, the code we have to handle this elsewhere is:

```ruby
GRADLE_RANGE_REGEX = /[\(\[].*,.*[\)\]]/.freeze

def self.parse_requirement_string(package_manager, string)
  if %w(nuget gradle maven).include?(package_manager) &&
     string.match?(GRADLE_RANGE_REGEX)
    return string
  end

 string.split(",").map(&:strip)
end
```

Reply:
@feelepxyz @jurre thanks for the feedback!

Just to be clear: should I update this PR to handle both styles, or is @jurre going to add a fix elsewhere in `dependabot-core`? I’m good either way 👍.

Reply:
It was pretty trivial to do, so I went ahead and got it started in `7845ad7`. _Not_ sharing code between package manager modules seems to be the convention, but let me know if anything should be DRY-er.

Reply:
Thank you @eager 🎉 

> Not sharing code between package manager modules seems to be the convention

It's definitely something we're trying to improve where it makes sense but for now I think this looks good 👍